v.example.html

```diff
@@ -1,24 +1,45 @@
 <h2>DESCRIPTION</h2>
 
-<em>v.example</em> is an example for a GRASS GIS addon
+<em>v.example</em> is an example for a GRASS GIS addon. Here comes a short
+description of the module.
+
+<p>
+New paragraph for more detailed description.
+
+<h2>NOTES</h2>
+
+Rather technical notes are here.
+
+<p>
+Description of the algorithm or method
+
+<p>
+Hints for usage, e.g. for large maps or a large number of input arguments.
 
 <h2>EXAMPLE</h2>
 
-<h3> Example 1</h3>
+<h3>Create grid for current region with grid box size of 1000 x 1000</h3>
 
 <div class="code"><pre>
 v.example output=tiles box=1000,1000
 </pre></div>
 
-<h3>Create grid for current region with grid box size of 1000 x 1000 and output basename "tiles" and extract tiles only for a given polygon "poly_aoi".</h3>
-
+<h3>Extract tiles only for a given polygon "poly_aoi"</h3>
 <div class="code"><pre>
-v.example output=tiles box=1000,1000 polygon_aoi=poly_aoi
+v.select ainput=tiles atype=area binput=poly_aoi btype=area operator=overlap output=tiles_aoi
 </pre></div>
 
+Image with the result
+<center>
+<img src="v_example.png" border=1><br>
+<i>Selected tiles</i>
+</center>
```

Comment:
```suggestion
<div align="center" style="margin: 10px">
<a href="v_example.png">
<img src="v_example.png" width="600" height="600" alt="v.example example" border="0">
</a><br>
<i>Figure: Vector map .... something.</i>
</div>
```

Following https://trac.osgeo.org/grass/wiki/Submitting/Docs#HTMLPages